lib/plugins.js

```diff
@@ -58,8 +55,10 @@ function pluginFunFactory(funName) {
       var pluginConf = this.pluginConfs[name];
       var pluginObj = this.pluginObjs[name];
       names.push(name);
-      promises.push((pluginObj[funName] || noop)(pluginConf));
+      promises.push(
+          (pluginObj[funName] || noop)(pluginConf, [].slice.call(arguments)));
```

Comment:
I don't think this is consistent with what you put in `plugins/README.md`.  Shouldn't it be `(pluginObj[funName] || noop).apply(this, [pluginConf].concat([].slice.call(arguments)))`?